### DISCUSSION TITLE
posts/deploying-loki-and-promtail-together-with-the-tig-stack/

### DISCUSSION THREAD

**Discussion by giscus** [Roxedus/nonsense.fyi#7] [Announcements]
# posts/deploying-loki-and-promtail-together-with-the-tig-stack/

How to set up Loki, Promtail, Telegraf and Grafana to visualize logs and system stats.

https://nonsense.fyi/posts/deploying-loki-and-promtail-together-with-the-tig-stack/

**Comment by Roxedus** [comment]
Comments!
